docs/base/conf.py

```diff
@@ -0,0 +1,58 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Elpi'
+copyright = '2022, Enrico Tassi'
+author = 'Enrico Tassi'
+
+# The full version, including alpha/beta/rc tags
+release = '1.15.2'
```

Comment:
I think we should read this from `git describe --exact-match` (or `"master"`), see the prev comment